### ISSUE TITLE
Support for SLURM?

### ISSUE THREAD

**Issue by jrudoler** [pyushkevich/ashs#7]
Software is great, our lab has had lots of success using it for our imaging pipeline. One additional feature that would be nice is support for SLURM instead of SGE. We've had problems with SGE not properly propagating errors, and not setting memory limits - SLURM has worked better and has more thorough open-source documentation.  How feasible would this be?

**Comment by pyushkevich** [comment]
Hi Joseph, I believe SLURM and LSF are already supported in the latest
version - please check on Github.
Paul

**Comment by jrudoler** [comment]
You are totally right! It does support both, and there's detailed instructions in the updated docstrings / help for the training functions. I was reading the online documentation [here](https://sites.google.com/view/ashs-dox/local-ashs/building-an-atlas-for-t1-mri?authuser=0) which doesn't mention anything about it. Thanks!
